docs/extras/advanced-variables/variable-schemas.md

```diff
@@ -7,7 +7,7 @@ Variable Schemas is a new DevCycle feature that lets you define allowable values
 
 ## Defining a Variable Schema
 
-To define a Variable Schema for a variable, navigate to the Schema section of the Variable Details page to select and define a schema for that particular variable. The variable must be unassociated in order to select and define a schema. 
+To define a Variable Schema for a variable, navigate to the Schema section of the Variable Details page to select and define a schema for that particular variable. Variable Schemas can be added to both associated or unassociated variables. 
```

Comment:
I would just drop the second sentence now because it isn't really adding additional information to someone new who might be reading this